ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessorTest.kt

```diff
@@ -226,6 +287,37 @@ class VariableProcessorTest {
         expectVariablesResult("{{ sub_offer_price }}", "", rcPackage = TestData.Packages.monthly)
     }
 
+    // endregion
+
+    // region sub_offer_price_2
+
+    @Test
+    fun `process variables processes sub_offer_price_2 for products with no intro price`() {
+        expectVariablesResult("{{ sub_offer_price_2 }}", "", rcPackage = TestData.Packages.annual)
+    }
+
+    @Test
+    fun `process variables processes sub_offer_price_2 for intro prices`() {
```

Comment:
Should this test be something like `process variables processes sub_offer_price_2 for intro prices when free trial available`? Since it should be null for intro prices when there is no free trial available I would say.

Reply:
Check [9ea9167](https://github.com/RevenueCat/purchases-android/pull/1313/commits/9ea916718705fa928c82e36f9d107346cf5113c9). I fixed tests. Names were weird or didn't make sense lol